sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.aest.controller.getApplicationDetails.e2e-spec.ts

```diff
@@ -0,0 +1,133 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../testHelpers";
+import {
+  createE2EDataSources,
+  E2EDataSources,
+  saveFakeApplication,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { EducationProgramOffering, OfferingIntensity } from "@sims/sims-db";
+import { getUserFullName } from "../../../utilities";
+import {
+  addDays,
+  getDateOnlyFormat,
+  getISODateOnlyString,
+} from "@sims/utilities";
+
+describe("ApplicationAESTController(e2e)-getApplicationDetails", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should get the student application details.", async () => {
```

Comment:
Please have a description for when stating the condition of test case. 

e.g `when the optional query parameter to load dynamic data is not passed.`

Reply:
updated